pkg/chain/ethereum/bitcoin_difficulty.go

```diff
@@ -187,7 +188,12 @@ func (bdc *BitcoinDifficultyChain) RetargetWithRefund(headers []*bitcoin.BlockHe
 	}
 
 	// Update Bitcoin difficulty via LightRelayMaintainerProxy.
-	_, err := bdc.lightRelayMaintainerProxy.Retarget(serializedHeaders)
+	_, err := bdc.lightRelayMaintainerProxy.Retarget(
+		serializedHeaders,
+		ethutil.TransactionOptions{
+			GasLimit: 310000,
```

Comment:
Instead of hardcoding, I'd take the estimation and add some % on top. Hardcoding the gas limit may stop working if EVM opcode call costs change.